cardano-api/src/Cardano/Api/LocalStateQuery.hs

```diff
@@ -208,7 +208,7 @@ queryLocalLedgerState
   => Network
   -> SocketPath
   -> Point blk
-  -> ExceptT LocalStateQueryError IO (Either LByteString (Ledger.LedgerState TPraosStandardCrypto))
+  -> ExceptT LocalStateQueryError IO (Either LByteString (Ledger.EpochState TPraosStandardCrypto))
 queryLocalLedgerState network socketPath point = do
```

Comment:
Do we want to rename `LedgerState` everywhere to `EpochState`, including the CLI command name? I think `LedgerState` is more descriptive than `EpochState`, so I left it as is.

Reply:
> I think LedgerState is more descriptive than EpochState, so I left it as is.

I agree.